src/forklift/core.py

```diff
@@ -65,12 +66,13 @@ def remove_temp_table(table):
                      exc_info=True)
             return (Crate.INVALID_DATA, e.message)
 
-        try:
-            remove_temp_table(crate.destination + reproject_temp_suffix)
-            has_changes = _has_changes(crate)
-        except:
-            log.warn('Exception thrown while checking for changes. Assuming that there are changes...', exc_info=True)
-            has_changes = True
+        # try:
+        #     remove_temp_table(crate.destination + reproject_temp_suffix)
+        #     has_changes = _has_changes(crate)
+        # except:
+        #     log.warn('Exception thrown while checking for changes. Assuming that there are changes...', exc_info=True)
+        #     has_changes = True
+        has_changes = True
         if has_changes:
```

Comment:
Doesn't look like this statement is necessary anymore because we are dropping `_has_changes()` (yea!). Should also remove the corresponding `else` block.

Also need to remove the `_has_changes` method declaration from the bottom of this file.